app/src/main/java/com/vero/tasky/agenda/presentation/editphoto/EditPhotoScreen.kt

```diff
@@ -0,0 +1,68 @@
+package com.vero.tasky.agenda.presentation.editphoto
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material.icons.filled.Delete
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.vero.tasky.R
+import com.vero.tasky.ui.theme.Inter600Size20
+import com.vero.tasky.ui.theme.headerText
+
+@Composable
+fun EditPhotoScreen(
+    uri: String,
```

Comment:
You could pass a Uri object instead of a String

And do the .path here

Mostly so you verify it´s a valid uri and not some random string

Reply:
Nvm. You are using Compose Navigation. So you can´t pass a Uri as parameter